packages/components/src/components/date-picker/date-picker.tsx

```diff
@@ -1,25 +1,63 @@
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, { type ReactElement } from 'react';
```

Comment:
```suggestion
import React from 'react';
```
please import React only and use ReactElement as `React.ReactElement`. please refer to import rules on this project in README.md line 120